solutions/swb-app/src/accountRoutes.ts

```diff
@@ -34,6 +36,16 @@ export function setUpAccountRoutes(router: Router, hostingAccountService: Hostin
     })
   );
 
+  router.post(
```

Comment:
That's a bit strange to have POST HTTP verb and get-* API name. Suggest changing to get and removing `get-` from route